common/queue/src/main/java/org/thingsboard/server/queue/kafka/TbKafkaSettings.java

```diff
@@ -149,6 +180,26 @@ public class TbKafkaSettings {
 
     private volatile AdminClient adminClient;
 
+    @PostConstruct
+    public void initConsumerPropertiesPerTopic() {
+        patchTopicAlias("tb_ota_package", buildTopicName(otaPackageTopic));
```

Comment:
Instead of doing all that - just add queue.kafka.consumer-properties-per-topic.**other-inline** - the user will be able to add their own consumer config if they change some topic.